packages/angular/src/app/modules/builder/utils/constants.ts

```diff
@@ -0,0 +1,3 @@
+// TODO remove hardcoded version, maybe a release tag?
+const ANGULAR_LATEST_VERSION = '1.3.49';
+const SCRIPT_ID = 'builder-wc-script';
```

Comment:
don't we need to export these and import them in the `builder-component.component.ts` file?

Reply:
yes indeed. Somehow my editor didnt complain at that time.

fixed it.